### PR TITLE
Fix issue: no login button in signup page🔧

### PR DESCRIPTION
**Pull request by Riteshgtm12**
#5 I have fixed the issue, and I also put the signup button on the login page, so now you can review from your side. If you are satisfied, you can merge the code.  after that you can close this issue

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -141,4 +141,13 @@ footer {
     max-width: 200px;
     max-height: 200px;
     border-radius: 12px;
-}
\ No newline at end of file
+}
+#login-link{
+    margin-top: 10px;
+  }
+  .login-btn{
```
Comment (omrajsharma) [inline src/App.css]: fix the identation

File: src/pages/LoginPage.jsx [hunk 2]
```diff
@@ -73,6 +74,9 @@ const LoginPage = () => {
             >
               Login
             </Button>
+            <div id="login-link">
```
Comment (omrajsharma) [inline src/pages/LoginPage.jsx]: please fix the formatting

File: src/pages/RegisterPage.jsx [hunk 3]
```diff
@@ -106,6 +107,9 @@ const RegisterPage = () => {
             >
               Sign Up
             </Button>
+            <div id="login-link">
```
Comment (omrajsharma) [inline src/pages/RegisterPage.jsx]: formatting required

File: src/App.css (hunk 1 above)
Comment (Riteshgtm12) [reply src/App.css]: indentation means?

File: src/pages/LoginPage.jsx (hunk 2 above)
Comment (Riteshgtm12) [reply src/pages/LoginPage.jsx]: ok sure i'll fix it

File: src/pages/RegisterPage.jsx (hunk 3 above)
Comment (Riteshgtm12) [reply src/pages/RegisterPage.jsx]: ok sure i'll fix it

File: src/App.css (hunk 1 above)
Comment (Riteshgtm12) [reply src/App.css]: Fixing Done You can check it again

File: src/pages/RegisterPage.jsx (hunk 3 above)
Comment (Riteshgtm12) [reply src/pages/RegisterPage.jsx]: FIXED IT. YOU CAN CHECK

File: src/pages/LoginPage.jsx (hunk 2 above)
Comment (Riteshgtm12) [reply src/pages/LoginPage.jsx]: FIXED IT. YOU CAN CHECK

**Review by omrajsharma — APPROVED** [review]

**Pull request merged**